### ISSUE TITLE
Breadcrumb A11y Pass

### ISSUE THREAD

**Issue by TestAccessibilityGIT** [microsoft/fluentui#6468]
Tracking all of the issues brought up with the 09/25 accessibility pass on Breadcrumb  control
Details on individual bugs can be found on [SharePoint](https://microsoft-my.sharepoint.com/:x:/r/personal/v-amaver_microsoft_com/_layouts/15/Doc.aspx?sourcedoc=%7Be132bb17-3dda-4610-b331-79ef456ea4c5%7D&action=default&uid=%7BE132BB17-3DDA-4610-B331-79EF456EA4C5%7D&ListItemId=1399&ListId=%7B395E1ECB-5002-440E-81C5-6A090E98077C%7D&odsp=1&env=prodbubble)

- [x] Bug 1: [Accessibility][Programmatic access]: Buttons must have discernible text."
- [x] Bug 2: [Accessibility][Keyboard navigation]: Landmarks must have a unique role or role/label combination(aria-label OR aria-labelledby)"
- [x] Bug 3: [Accessibility][Supporting the platform]: Breadcrumb button Controls are not visible on mouse focus in high contrast white/black themes.

Duplicates:
<s>
- Bug 4: [Accessibility][Forms and validation]: Group labels inside dropdowns are not associated with its corresponding control.
- Bug 5: [Accessibility][Usable]: Screen does not scroll down along with keyboard focus in back navigation.
- Bug 6: [Accessibility][Screen reader]: Narrator focus moves twice on the first element of context menu in scan mode. 
- Bug 7: [Accessibility][Screen reader]: Narrator focus moves onto the separator line in scan mode. 
- Bug 8: [Accessibility][Screen reader]: In Scan mode user is not able to close the 'Contextual menu' using 'Esc key'.
- Bug 9: [Accessibility][Screen reader]: Narrator focus moves onto hidden elements  in scan mode in backward navigation. 
- Bug 10: [Accessibility][Usable]: Scan mode focus is not visible on 'Grid view', 'More option' and 'Info button'. 
- Bug 11: [Accessibility][Visual requirements]: 'Tooltip' is not defined for 'More option' control. 
</s>

Component: Breadcrumb
@micahgodbolt

**Comment by natalieethell** [comment]
@lynamemi fixed #3 in this PR: https://github.com/OfficeDev/office-ui-fabric-react/pull/4728. I'm still seeing the issue on the website, but it looks fine when I run it locally. 

Website
![image](https://user-images.githubusercontent.com/14134000/47466240-31ce9d00-d7a5-11e8-8aaf-876081b87cef.png)

localhost
![image](https://user-images.githubusercontent.com/14134000/47466255-457a0380-d7a5-11e8-97cc-d589b16e0276.png)


Emily, do you know why we would still be seeing this issue on the website? It seems like the changes you made 6 months ago should have propagated by now.

**Comment by natalieethell** [comment]
Bugs #4 - 11 are issues with other Fabric controls and have already been reported in separate issues for those controls. I'll cross them out here since they're duplicates.

**Comment by lynamemi** [comment]
@natalieethell It looks like another case of style clashes with the mwf stylesheets we get from using the UHF header. We struggle with that issue all over the website with links - the current working solution is to target the website's styles more specifically which is not ideal.  This is what I get when I inspected a focused breadcrumb link:
![image](https://user-images.githubusercontent.com/16619843/47594211-260ee200-d92f-11e8-89a5-9f178c60db74.png)

**Comment by Jahnp** [comment]
Yup. That beastly selector is the cause of this pain. I've unfortunately had to re-introduce it locally on the site in #6960 until we can push a better fix into MWF (or remove it from the site altogether)

**Comment by micahgodbolt** [comment]
wow

**Comment by natalieethell** [comment]
Thank you @Jahnp!

**Comment by natalieethell** [comment]
Just checked this on the site, looks good!

**Comment by 999-Dev** [comment]
Below bugs are fixed on Test Enviourment. Hence closing those bug.
**Bug 1, Bug 2, Bug 4, Bug 5, Bug 6, Bug 7, Bug 8, Bug 9.**

**Test Environment:** 
OS: Version: RS5_Prerelease(1903 OS Build 18998.1) 
URL: https://developer.microsoft.com/en-us/fabric#/controls/web/breadcrumb
Edge Browser Version: 
Microsoft Edge: 44.18998.1.0
Microsoft EdgeHTML: 18.18998
